s/strimzi-kafka-oauth/redhat_ubi8/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Hi Amir, 
Can we make use of this Container Image instead `registry.access.redhat.com/ubi8/openjdk-8` ?

Reply:
I wish we could, it is corrupted for both x86 and Power. I tried it last week :(